### DISCUSSION TITLE
Lunr Language Support - Sponsorship

### DISCUSSION THREAD

**Discussion by iamcarbon** [bleroy/lunr-core#21] [General]
If anyone is interested in helping port the lunr-launguage library (stemmers, filters, stopwords, etc) to enable multiLanguage support, would be very keen on posting a sponsorship toward the effort.

It looks like a fair amount of work, and could possibility be implemented one language at a time.

https://github.com/MihaiValentin/lunr-languages

**Comment by bleroy** [comment]
Absolutely, that was my next order of business, but I've been lacking time. What's nice is that this looks like it can be easily parallelized.
